webapp/crashstats/crashstats/tests/test_jinja_helpers.py

```diff
@@ -139,36 +140,6 @@ def _create_report(self, **overrides):
     def _extract_query_string(self, url):
         return parse_qs(urlsplit(url).query)
 
-    def _create_frame(
```

Comment:
These kinds of methods make it difficult to read and verify tests because it's not clear how they're setting up the test scenario. Since I moved most of the tests that these were used in to `test_libbugzilla.py`, I fixed the ones that remained in here to not uses these methods and then removed the methods.